{"name": "Авторизация под несуществующим пользователем", "status": "passed", "description": "Проверка, что с несуществующими данными возвращается ошибка 404", "steps": [{"name": "Логин курьера", "status": "passed", "parameters": [{"name": "payload", "value": "{'login': 'nonexistentuser999', 'password': 'nonexistentpass999'}"}], "start": 1761487122329, "stop": 1761487122980}], "start": 1761487122329, "stop": 1761487122981, "uuid": "eb766265-529e-4ad7-a20e-9b7ccd460d3c", "historyId": "90107893f32250a09a6946cab84ca29a", "testCaseId": "90107893f32250a09a6946cab84ca29a", "fullName": "tests.test_courier_login.TestCourierLogin#test_login_non_existent_user_fails", "labels": [{"name": "suite", "value": "Логин курьера"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "Astti-Legion"}, {"name": "thread", "value": "132044-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}